examples/ibm-private-dns/main.tf

```diff
@@ -121,3 +121,22 @@ data "ibm_dns_resource_records" "test-res-rec" {
   instance_id = ibm_resource_instance.test-pdns-instance.guid
   zone_id = ibm_dns_resource_record.test-pdns-resource-record-a.zone_id
 }
+
+resource "ibm_dns_glb_pool" "test-pdns-pool-nw" {
+		name = "testpool"
+		instance_id = ibm_resource_instance.test-pdns-instance.guid
+		description = "new test pool"
+		enabled="true"
```

Comment:
it is bool attribute. remove double quote

Reply:
handled